dbus/org.freedesktop.ratbag_devel1.conf.in

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?> <!--*-nxml-*-->
+<!DOCTYPE busconfig PUBLIC
+        "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+        "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+
+<busconfig>
+
+        <policy user="root">
+                <allow own="org.freedesktop.ratbag_devel1_@ratbagd_sha@"/>
+                <allow send_destination="org.freedesktop.ratbag_devel1_@ratbagd_sha@"/>
+                <allow receive_sender="org.freedesktop.ratbag_devel1_@ratbagd_sha@"/>
+        </policy>
+
+	<!-- allow everyone to talk to the daemon -->
+        <policy context="default">
+                <allow send_destination="org.freedesktop.ratbag_devel1_@ratbagd_sha@"/>
+                <allow receive_sender="org.freedesktop.ratbag_devel1_@ratbagd_sha@"/>
+        </policy>
+
+</busconfig>
```

Comment:
This should have a warning in the form of "FOR DEVELOPMENT ONLY. If you don't know where this file came from, it can be safely removed."